### PR TITLE
Cust 1756 display recent unread user alerts in user menu

### PR DESCRIPTION
**Pull request by darushka**
This PR implements the display of recent unread alerts in User Menu dropdown.

### CODE REVIEW

**Comment by jsmylnycky** [comment]
:+1:

File: src/controllers/alert/alert.controller.js [hunk 1]
```diff
@@ -0,0 +1,366 @@
...
+        if(ids.length) {
+          this.mergeWithAlertEvents(ids);
+        }
+      }, (error) => {
+        this.$log.error(error);
+      });
+  }
+
+  getItemAtIndex(index) {
+    let pages = Math.ceil(this.alerts[this.tabs[this.selectedIndex].category].count / this.query.limit);
```
Comment (rynomars) [inline src/controllers/alert/alert.controller.js]: to be on the safe side, you might want to check if this.query.limit === 0 before doing the division.

**Comment by rynomars** [comment]
:thumbsup:

**Comment by jsmylnycky** [comment]
@darushka please merge squash

**Comment by darushka** [comment]
Merged squash https://github.com/peachworks/ngPeach/commit/dd37ae1c206ecec789d98d69827af3d93d8b3692

**Comment by jsmylnycky** [comment]
Released v0.6.0

**Pull request closed without merging**